### PR TITLE
feat:cart page added

### PR DESCRIPTION
**Pull request by Developers-codz**
Added Cart page.
It is fully responsive 

Do review and give your valuable feedback. Thank you :)

### CODE REVIEW

**Comment by devsurajrai** [comment]
Good Job. Looks great.

**Review by devsurajrai — APPROVED** [review]
Well Done. Everything looks good to me.

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        <div class="cart-left-pane">
+           <div class="cart-items">
+               <img src="./assets/item2.jpg" alt="product">
+               <div class="cart-product-detail">
+                   <h3 class="text-vibrant">Wooden Basket</h3>
+                   <p class="text-secondary ">Environment Friendly Baskets</p>
+                   <p class="item-price font3 mb-lg">₹499 only</p>
+                   <div class="quantity mb-lg">
+                       <button>-</button>
+                       <input type="text" disabled value="2">
```
Comment (ApurvaSawant11) [inline cart.html:99]: The `quantity` div items are not aligned properly. Display them as flex and align them. Or match the height of the buttons and input field

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me!

File: cart.html (hunk 1 above)
Comment (Developers-codz) [reply cart.html:99]: Done!

**Review by arjun-computer-geek — APPROVED** [review]
Clean code and ui both good to go for merge

**Pull request merged**